mms/tests/integration_tests/mms_mxnet_model_zoo_integ_test.py

```diff
@@ -0,0 +1,46 @@
+import json
+import os
+import shutil
+import subprocess
+import tarfile
+from threading import Thread
+import pytest
+import sys
+import time
+import signal
+import test_utils as utils
+try:
+    from urllib2 import urlopen, URLError, HTTPError
+except:
+    from urllib.request import urlopen, URLError, HTTPError
+
+from mms import mxnet_model_server
+
+
+
+
+### models from onnx-mxnet model zoo
+mxnet_model_URLs = {
+    ##TODO: check why these two models are failing
```

Comment:
is there a GH issue for this? change comment to reference issue number

Reply:
created an issue and referenced